backend/src/services/ai/ai-quota.service.ts

```
@@ -0,0 +1,167 @@
+    );
+    return (result.rowCount ?? 0) > 0;
+  }
+
+  private mapRow(row: Record<string, unknown>): QuotaConfig {
+    return {
+      id: row.id as string,
+      userId: row.user_id as string | null,
+      maxRequestsPerDay: row.max_requests_per_day as number | null,
+      maxTokensPerDay: row.max_tokens_per_day as number | null,
+      maxTokensPerMonth: row.max_tokens_per_month as number | null,
+      maxSpendUsdPerMonth: row.max_spend_usd_per_month != null ? parseFloat(String(row.max_spend_usd_per_month)) : null,
```

ESLint

[error] Replace `·row.max_spend_usd_per_month·!=·null·?·parseFloat(String(row.max_spend_usd_per_month))` with `⏎········row.max_spend_usd_per_month·!=·null⏎··········?·parseFloat(String(row.max_spend_usd_per_month))⏎·········`

(prettier/prettier)